templates/CRM/common/civicrm.settings.php.template

```diff
@@ -261,7 +261,7 @@ if (!defined('CIVICRM_UF_BASEURL')) {
  */
 if (!defined('CIVICRM_SMARTY_AUTOLOAD_PATH') && getenv('CIVICRM_SMARTY_AUTOLOAD_PATH') === FALSE) {
   // enable by default for dev & demo sites for now - will soon enable by default for all new installs.
-  if (strpos(CIVICRM_UF_BASEURL, 'localhost') !== FALSE || strpos(CIVICRM_UF_BASEURL, 'demo.civicrm.org') !== FALSE) {
+  if (CIVICRM_UF === 'UnitTests' || strpos(CIVICRM_UF_BASEURL, 'localhost') !== FALSE || strpos(CIVICRM_UF_BASEURL, 'demo.civicrm.org') !== FALSE) {
```

Comment:
I think we should add in here `|| strpos(CIVICRM_UF_BASEURL, 'nip.io') !== FALSE`

cc @demeritcowboy

Reply:
@seamuslee001 yeah - I guess that would work - I grabbed the string `http://build` which I think would also do it & *might* capture more buildkit installs? (we are pretty close to switching 5 to being the default so a bit greedy is OK I think)